modules/gitops-release-notes-1-2.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-2_{context}"]
+= Release notes for {gitops-title} 1.2
+
+{gitops-title} 1.2 is now available on {product-title} 4.7 and 4.8.
+
+[id="support-matrix-1-2_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {gitops-title} 1.2
+| Argo CD
+| GA
+| Argo CD ApplicationSet
+| TP
+| {gitops-title} Application Manager (kam)
+| TP
+| {gitops-title} Service
+| TP
+|===
+
+[id="new-features-1-2_{context}"]
+== New features
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.2:
+
+* If you do not have read or write access to the openshift-gitops namespace, you can now use the `DISABLE_DEFAULT_ARGOCD_INSTANCE` environment variable in the GitOps Operator and set the value to `TRUE` to prevent the default Argo CD instance from starting in the `openshift-gitops` namespace.
+* Resource requests or limits are now configured in Argo CD workloads.  Resource quota is enabled in the `openshift-gitops` namespace. As a result, out-of-band workloads deployed manually in the openshift-gitops namespace must be configured with resource requests or limits and the resource quota may need to be increased.
+* Argo CD auth is now integrated with Red Hat SSO and it is automatically configured with OpenShift 4 Identity Provider on the cluster. This feature is disabled by default.  To enable Red Hat SSO, add SSO configuration in ArgoCD CR as shown below. Currently,`keycloak` is the only supported provider.
```

Comment:
auth/authentication
in ArgoCD CR/ in the `ArgoCD` CR 
ArgoCD must be in back ticks and the article 'the' must be added before it, please.